components/gitops/.tekton/trigger-template.yaml

```diff
@@ -34,4 +34,4 @@ spec:
           - name: workspace
             persistentVolumeClaim:
               claimName: app-studio-default-workspace
-            subPath: gitops-service-$(tt.params.git-revision)
+            subPath: pv-gitops-service/$(tt.params.git-revision)
```

Comment:
do not replace `-` by `/` and then it will work with the current cleanup script.